### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jerryshia**
Fix issue #35775 

@rohit-ganguly , @lmazuel , @lfraleigh , @mayurid , @sarkar-rajarshi for notification.

### CODE REVIEW

**Review by rohit-ganguly — APPROVED** [review]

**Comment by v-xuto** [comment]
@rohit-ganguly Could you help merge this PR? Thanks a lot.

**Comment by lmazuel** [comment]
Reading the EG code, I don't believe this is the right fix. Closing, and having the discussion on the issue.

**Comment by lmazuel** [comment]
Re-opening with a commit suggestion instead of just closing

File: sdk/communication/azure-communication-jobrouter/README.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ from azure.eventgrid import EventGridEvent
 import json
 
 ## deserialize payload into a list of typed Events
-events = [EventGridEvent.from_json(json.loads(msg)) for msg in payload]
+events = [EventGridEvent.from_json(json.dumps(msg)) for msg in payload]
```
Comment (lmazuel) [inline sdk/communication/azure-communication-jobrouter/README.md:198]:
```suggestion
events = [EventGridEvent.from_json(msg) for msg in payload]
```

There is a lack of clarity into what `msg` is coming from, so in doubt we should keep it simple. Also, `from_json` will expect a string, or a commplex type from store/eventhub

**Comment by jerryshia** [comment]
@lmazuel I have updated by your comments, please review.

**Comment by v-xuto** [comment]
@lmazuel We have updated by your comments, please re-review. Thanks a lot.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**